### DISCUSSION TITLE
The lack of "search" in the list of commands is killing me lol.

### DISCUSSION THREAD

**Discussion by OnecaresDen** [Sehelitar/Kick.bot#60] [General]
![obraz](https://github.com/Sehelitar/Kick.bot/assets/174027255/560c5867-8fcf-4d0d-aba2-a1ad987a6dbc)
I have 1047 commands and with the "cooldown" version of the command it's simply doubled. I really need a search bar so I  can quickly write what i'm searching for. Also another one - what is the command to get the avatar of new follower? Thank you for your help and i'm really grateful for the plugin!
Edit: Maybe you can make the twitch command also work for kick by default (togglable or something like that)

**Comment by Sehelitar** [comment]
Hey! 👋 
Believe me, if I could have done better, I would have 😄 
Streamer.bot doesn't give us this kind of liberty with the software's interface. So unless that changes, this is the only way it can work for now.

As for user's avatars, you can use the GetUserInfos method and the ``targetUserProfileImageUrl`` argument, but this one will only be set if the user actually has a profile picture. Kick doesn't send a "default" picture back like Twitch does. (See #61 )

**Reply by OnecaresDen** [reply]
Thank you very much bro!

**Comment by Sehelitar** [comment]
[A new update](https://github.com/Sehelitar/Kick.bot/releases/latest) (v0.4.1) is available and provide a link to the official default profile picture if the user didn't set one.
